chatterbox/src/Main/Header/index.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import './index.css';
+import Oval from '../../assets/Oval.png';
+import Oval_2 from '../../assets/Oval_2.png';
```

Comment:
Hmm, these Oval images seem more like circles. Also watch your casing of the file names. Lower case with dashes for file names are better, but it depends on what you decided for your filenames.

Reply:
changed Oval to lower case images

Reply:
changed Oval to lower case images